.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+name: Netengine CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    name: Python==${{ matrix.python-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Upgrade python system packages
+      run: pip install -U "pip==20.2.4" wheel setuptools
```

Comment:
Can you try removing `==20.2.4`? Maybe the new dependency resolution of pip will not cause issue here.